Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -1254,18 +1254,18 @@ static bool isNumber(const CSSValue& value, double number, CSSUnitType type)
 
 static bool isNumber(const RectBase& quad, double number, CSSUnitType type)
 {
-    return isNumber(quad.top(), number, type)
-        && isNumber(quad.right(), number, type)
-        && isNumber(quad.bottom(), number, type)
-        && isNumber(quad.left(), number, type);
+    return isNumber(Ref { quad.top() }.get(), number, type)
```

Comment:
This is not good. The point of these functions like `isNumber` is to create a simple caller interface so functions like this are brief and easy to read. If callers have to keep doing `Ref` we should change the idiom. The idea is that these inline functions are "free" and easy to use and result in readable code.

I understand that we are just doing what the static analyzer says but it's going against why we wrote the code the way we did.